CHANGELOG.md

```diff
@@ -1,29 +1,67 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
-See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
+See the [change log guidelines](https://github.com/halestudio/hale/compare/5.0.1...5.1.0) for information on how to structure the file.
+
+## [Unreleased](https://github.com/halestudio/hale/compare/5.0.1...5.1.0)
+
+## [5.1.0]
 
-## [Unreleased]
+### Added
+
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Added support for importing data from JSON files, enhancing the versatility of data import options.
+- Introduced support for loading JSON schema, providing users with greater flexibility in handling JSON data structures.
+- Improve the export to Shapefiles including feature types with no geometry
+- Support adding multiple selections of feature types to be exported to CSV
+- Replace default map with OpenStreetMap
+- Add Boolean and Java Pattern class to Groovy restriction whitelist
+- Support up to 20 decimal places for source data geometries, ensuring heightened precision and accuracy in the exported data. 
```

Comment:
Again the question what change this is related to? I assume it is only related to Json? And as a change only related to the Json/GeoJson export, where before a limit of 7(?) was used.

I would maybe rephrase it to

"When exporting data as Json or GeoJson, now up to 20 decimal places are retained in Geometry coordinates"